README.md

```diff
@@ -2,7 +2,15 @@
 
 A scraper for your documentation website that indexes the scraped content into a MeiliSearch instance.
 
+[MeiliSearch](https://github.com/meilisearch/MeiliSearch) is a powerful, fast, open-source, easy to use and deploy search engine. Both searching and indexing are highly customizable. Features such as typo-tolerance, filters, and synonyms are provided out-of-the-box
+
+This scraper is used in production and crawls the [MeiliSearch documentation](https://docs.meilisearch.com/) on each deployment.
```

Comment:
To be honest I wrote that without thinking it could have a difference 😂
I found out this difference: https://www.skylynapps.com/scrapemate/learn/help/general/difference-between-crawler-and-scraper.php
Still not really clear haha
I'm going to update the docs then.